app/views/projects/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h2>Project: <%= @project.name %></h2>
+
+<p>Material: <%= @project.material %></p>
+<p>Challenge Theme: <%= @project.challenge.theme %></p>
+<p>Number of Contestants: <%= @project.contestants.count %></p>
```

Comment:
I would build a method in your `project` model that does the math here. So something like 
```ruby 
def num_contestants
 contestants.count
end
```
then call `.num_contestants` on your `@project` on line 5.